vllm/entrypoints/openai/tool_parsers/hermes_tool_parser.py

```diff
@@ -1,6 +1,13 @@
 import json
 import re
 from typing import Dict, List, Sequence, Union
+try:
+    from json_repair import json_repair
+except ImportError:
+    # might remove codes below by adding json_repair in requirement.txt
+    import subprocess
+    subprocess.check_call(["pip", "install", "json_repair"])
+    from json_repair import json_repair
```

Comment:
Let's not install things automatically. Can you add this to the requirements file instead?

Reply:
> Let's not install things automatically during runtime. Can you add this to the requirements file instead?

I noticed there are a lot of requirements files, and I'm not familiar with their specific uses. It might need some assistance with it.

Reply:
The base dependencies of vLLM are contained in `requirements-common.txt`. Since tool parsing is applicable to all backends, you can add the dependency there.